client/my-sites/customer-home/cards/tasks/webinars/index.jsx

```diff
@@ -24,11 +24,11 @@ const Webinar = () => {
 			actionOnClick={ () => {
 				window.open( 'https://wordpress.com/webinars/', '_blank' );
 			} }
-			illustration={ webinarIllustration }
+			illustration={ webinarsIllustration }
 			timing={ 2 }
 			taskId="webinar"
```

Comment:
Spotted it, I think this should be `webinars` vs `webinar` to make the pref stick